doc/src/Examples.dox

```diff
@@ -109,9 +109,8 @@
  *
  * The API middleware provides assorted short-circuit validation
  * optimisations that can reduce the number of inter-language hops
- * required. See @ref
- * openassetio.managerAPI.ManagerInterface.ManagerInterface.info
- * "ManagerInterface.info" and the `kField_EntityReferencesMatchPrefix`
+ * required. See @fqref{managerAPI::ManagerInterface::info}
+ * "ManagerInterface::info" and the `kField_EntityReferencesMatchPrefix`
```

Comment:
[question] Should this PR also migrate the relevant constants to C++?  Or can we leave that to a separate issue?

Reply:
We can do that as a separate issue I think, but should do it now.

Reply:
Added https://github.com/TheFoundryVisionmongers/OpenAssetIO/issues/354